client/payment-methods/constants.ts

```diff
@@ -53,4 +53,13 @@ export const PAYMENT_METHOD_TITLES = {
 	wechat: __( 'WeChat', 'woocommerce-payments' ),
 };
 
+// eslint-disable-next-line @typescript-eslint/naming-convention
+export const BNPL_COUNTRY_CURRENCY_MAP: Record< string, string > = {
+	AU: 'AUD',
+	CA: 'CAD',
+	GB: 'GBP',
+	NZ: 'NZD',
+	US: 'USD',
+};
+
```

Comment:
Do we still need this block in JS if we get the currency from PHP?

Reply:
No, it is not needed anymore. Removed in [ecea42c](https://github.com/Automattic/woocommerce-payments/pull/6920/commits/ecea42c319eae7a7ee1d89c6b0a23089dc5f1e0e) thanks!